public/app/features/dashboard/components/DashboardPermissions/AccessControlDashboardPermissions.tsx

```diff
@@ -11,15 +11,7 @@ interface Props {
 }
 
 export const AccessControlDashboardPermissions = ({ dashboard }: Props) => {
-  const canListUsers = contextSrv.hasPermission(AccessControlAction.OrgUsersRead);
```

Comment:
question: how difficult would it be to perform the same check here as what you've added for the backend?